main.py

```diff
@@ -16,10 +22,13 @@ def login():
     if 'name' not in data.keys() or 'password' not in data.keys():
         return 'Name or/and password are missing.', 400
     
-    if data['name'] == 'denchik' and data['password'] == 'foobar':
-        return 'Correct name and password.', 200
-    
-    return 'Wrong name or password.', 403
+    if data['name'] not in credit.keys():
```

Comment:
Forgot to mention it in the previous review, `X in Y.keys()` is equivalent to `X in Y` if `Y` for a dict `Y`

Reply:
done.